lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -990,6 +991,22 @@ instance Buildable (EraInfo EpochNo) where
         boundF (Just e) = " from " <> build e
         boundF Nothing = " <not started>"
 
+data MinimumUTxOValue
+    -- | In Shelley, tx outputs could only be created if they were larger than
+    -- this `MinimumUTxOValue`.
+    = MinimumUTxOValue Coin
+
+    -- | With Alonzo, `MinimumUTxOValue` is replaced by a ada-cost per word of
+    -- the serialized output.
```

Comment:
"the serialized output" -> technically wrong. The ledger assumed a fixed length for `Coin` and `Address` values.